Tests/LinuxMain.swift

```diff
@@ -1,6 +1,12 @@
 import XCTest
+import MongoSwift
 import MongoSwiftTests
 
+/* Ensure libmongoc is initialized. Since XCTMain never returns, we have no
+ * opportunity to cleanup libmongoc (either explicitly or with a deinit). This
+ * may appear as a memory leak. */
+MongoSwift.initialize()
```

Comment:
Since Swift won't actually deinit a singleton (see: [SR-2500](https://bugs.swift.org/browse/SR-2500)), we need to explicitly call `mongoc_cleanup()` from a separate context. Two options as I see it:

 * Ask users to call an explicit function, as I'm doing here.
 * Make MongoSwift an instantiable class, which constructs the singleton (i.e. calls `mongoc_init()`) in `MongoSwift.init` and calls `mongoc_cleanup()` from `MongoSwift.deinit` (note: the singleton is never deinitialized).

I'm not a fan of the "instantiable class" approach as it puts the onus on the user to keep a MongoSwift object alive for the duration of their application. Failure to do so could result in a premature call to `mongoc_cleanup()`, which could cause curious failures and other headaches for users.